hexrd/ui/utils.py

```diff
@@ -244,3 +245,13 @@ def __init__(self, function):
     def __get__(self, obj, type=None) -> object:
         obj.__dict__[self.name] = self.function(obj)
         return obj.__dict__[self.name]
+
+
+@contextmanager
+def exclusions_off(plane_data):
+    prev = plane_data.exclusions
+    plane_data.exclusions = [False] * len(plane_data.exclusions)
```

Comment:
@psavery , setting `plane_data.exclusions = None` achieves the same result and may be "cleaner".

Reply:
Sounds good, I'll switch it.